### PR TITLE
PFDR-82: output status & error instead of json dump of queue item

### PR DESCRIPTION
**Pull request by aelkiss**
Needed to re-work some of the specs so that things worked as expected here (see PFDR-79 for the particular bug the previous fixtures were running into)

### CODE REVIEW

File: lib/uploader.rb [hunk 1]
```diff
@@ -40,9 +40,12 @@ def bag_id
 
   def print_result(qitem_result)
     if qitem_result["status"] == "DONE"
-      pp client.get(qitem_result["bag"])
+      puts "#{external_id} uploaded successfully"
+      true
```
Comment (botimer) [inline lib/uploader.rb]: I'm thinking we will want to move these `puts` calls over to using Logger. A logger bound to `STDOUT` and available via service locator throughout the app as the way to do output is pretty lightweight and flexible, accommodating levels/timestamps and making it easy to change formatting/destination decisions centrally.

That enhancement doesn't block this work -- we can make a separate issue for it.

File: spec/uploader_spec.rb [hunk 2]
```diff
@@ -3,44 +3,78 @@
...
-      expect(rsyncer).to receive(:upload).with(request["upload_link"])
-      subject.upload
+    context "when bag validation succeeds" do
+      it "uploads the bag" do
+        expect(rsyncer).to receive(:upload).with(request["upload_link"])
+        subject.upload
+      end
+
+      it "prints a success message" do
+        expect { subject.upload }.to output(/#{request["external_id"]}.*success.*/).to_stdout
```
Comment (botimer) [inline spec/uploader_spec.rb]: Extraneous trailing `.*`

File: spec/uploader_spec.rb [hunk 3]
```diff
@@ -3,44 +3,78 @@
...
     end
+
+    context "when bag validation fails" do
+      let(:queue_item) do
+         { status: "ERROR",
+           error: "something went wrong\nhere are the details" }
+      end
+
+      it "prints an error message" do
+        expect { subject.upload }.to output(/#{request["external_id"]}.*failure.*/).to_stdout
```
Comment (botimer) [inline spec/uploader_spec.rb]: Extraneous trailing `.*`

File: spec/uploader_spec.rb [hunk 4]
```diff
@@ -3,44 +3,78 @@
...
+      it "returns true" do 
+        expect(subject.upload).to be true
+      end
     end
+
+    context "when bag validation fails" do
+      let(:queue_item) do
+         { status: "ERROR",
+           error: "something went wrong\nhere are the details" }
+      end
```
Comment (botimer) [inline spec/uploader_spec.rb]: Formatting is a little funky here.

**Comment by aelkiss** [comment]
Thanks - I agree eventually we'll want to change the CLI to use a logger of some variety, but there's no requirement for that right now.

**Pull request merged**